study-builder/studies/rarex/emails/consent-assent-completed.html

```diff
@@ -65,98 +65,93 @@
+                You may return to the form in the DCP at any time if you wish to
+                update your contact information or to update your child/your
+                family members' health research data.
+            </p>
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                To update your data sharing preferences, you can return to the
+                Data Sharing Preference Agreement.
+            </p>
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                For your convenience, we have attached a copy of your signed parental consent form.
+            </p>
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                Please contact us at
+                <a href="mailto:support@rare-x.org" target="_blank" style="color: #26355c; text-decoration: none; cursor: pointer;">
+                    support@rare-x.org</a>
+                if you have any questions or concerns.
+            </p>
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                The information you submitted will be stored in the secure
+                RARE-X Data Collection Program. If you would like your
+                information deleted from our database at any time, now or in
+                the future, please email us at
+                <a href="mailto:support@rare-x.org" target="_blank" style="color: #26355c; text-decoration: none; cursor: pointer;">
+                    support@rare-x.org.</a>
+                We are so grateful for your participation - thank you again!
+            </p>
+        </td>
+    </tr>
+
+    <tr style="width: 540px;">
+        <td style="padding: 20px 0 80px 0;">
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                Helping Your Data Power Progress,
+            </p>
+
+            <p style="font-family: 'Roboto', sans-serif; text-align: left; font-weight: 400; font-size: 16px; line-height: 24px;">
+                The RARE-X Team
+            </p>
+        </td>
+    </tr>
+
+    <tr style="width: 540px;">
+        <td style="padding: 50px 0 0 0;">
+            <div data-role="module-unsubscribe" class="module" role="module" data-type="unsubscribe" style="padding:16px 16px 16px 16px; text-align:Center;" data-muid="4e838cf3-9892-4a6d-94d6-170e474d21e5">
+                <a href="{{{unsubscribe_preferences}}}" target="_blank" class="Unsubscribe--unsubscribePreferences" style="font-family: 'Roboto', sans-serif; text-decoration: none; font-size:12px; line-height:20px; color:#444444;">
+                    Unsubscribe
```

Comment:
Not sure if it was already mentioned, but SendGrid's "unsubscribe" feature will not stop Pepper from trying to send emails to participant. I don't know what this "unsubscribe" actually does -- maybe sendgrid will drop the emails and will not send. But on Pepper side, we track stuff like user_profile's `do_not_contact`.